components/public-api/gitpod/v1/envvar.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service EnvironmentVariableService {
+  // ListEnvironmentVariables returns all environment variables for the
+  // authenticated user.
+  //
+  // +return NOT_FOUND User does not have access to a workspace with the given
+  // ID +return NOT_FOUND Workspace does not exist
+  rpc ListEnvironmentVariables(ListEnvironmentVariablesRequest)
+      returns (ListEnvironmentVariablesResponse) {}
+
+  // UpdateEnvironmentVariable updates an environment variable for the
+  // authenticated user.
+  //
```

Comment:
Should we have `CreateEnvironmentVariable`?

It should also have oranization_id?

Reply:
why we need organization_id? old api doesn't have it, do you mean project_id?

Reply:
> Should we have CreateEnvironmentVariable?

Not sure current api does update or create at the same time, maybe as a follow up

Reply:
Sorry, I mean configuration_id. Project are named Configuration in gRPC Api.